content/docs/guides/logical-replication-fivetran.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Replicate data with Fivetran
+subtitle: Learn how to replicate data from Neon with Fivetran
+enableTableOfContents: true
+isDraft: false
+updatedOn: '2023-12-21T15:11:12.212Z'
+---
+
+Neon's logical replication feature allows you to replicate data from your Neon Postgres database to external destinations. 
+
+[Fivetran](https://fivetran.com/) is an automated data movement platform that helps you centralize data from disparate sources, which you can manage directly from your browser. Fivetran extracts your data and loads it into your data destination.
+
+In this guide, you will learn how to define a Neon Postgres database as a data source in Fivetran so that you can replicate data to one or more of Fivetran's supported destinations.
+
+## Prerequisites
+
+- A [Fivetran account](https://fivetran.com/)
+- A [Neon account](https://console.neon.tech/)
+
+## Enable logical replication in Neon
+
+<Admonition type="important">
+Enabling logical replication modifies the Postgres `wal_level` configuration parameter, changing it from `replica` to `logical` for all databases in your Neon project. Once the `wal_level` setting is changed to `logical`, it cannot be reverted. Enabling logical replication also restarts all computes in your Neon project, meaning active connections will be dropped and have to reconnect.
+</Admonition>
+
+To enable logical replication in Neon:
+
+1. Select your project in the Neon console.
+2. On the Neon **Dashboard**, select **Settings**.
+3. Select **Beta**.
+4. Click **Enable** to enable logical replication.
+
+You can verify that logical replication is enabled by running the following query from the [Neon SQL Editor](/docs/get-started-with-neon/query-with-neon-sql-editor):
+
+```sql
+SHOW wal_level;
+ wal_level 
+-----------
+ logical
+```
+
+## Create a Postgres role for replication
+
+It is recommended that you create a dedicated Postgres role for replicating data. The role must have the `REPLICATION` privilege. The default Postgres role created with your Neon project and roles created using the Neon Console, CLI, or API are granted membership in the [neon_superuser](/docs/manage/roles#the-neonsuperuser-role) role, which has the required `REPLICATION` privilege.
```

Comment:
Is this a general Postgres recommendation, or Neon? If Neon, consider:
"Neon recommends..." or "We recommend..."
I'd probably go with "We recommend..." regardless actually. In favor of the move towards "opinionated" advice.

Reply:
changed as suggested